x-pack/plugins/rule_registry/server/lib/get_is_kibana_request.ts

```diff
@@ -8,7 +8,7 @@ import type { Headers } from '@kbn/core/server';
 
 /**
  * Taken from
- * https://github.com/elastic/kibana/blob/ec30f2aeeb10fb64b507935e558832d3ef5abfaa/x-pack/plugins/spaces/server/usage_stats/usage_stats_client.ts#L113-L118
+ * https://github.com/elastic/kibana/blob/ec30f2aeeb10fb64b507935e558832d3ef5abfaa/x-pack/platform/plugins/shared/spaces/server/usage_stats/usage_stats_client.ts#L113-L118
```

Comment:
Original link should remain, as it references a specific commit.
```suggestion
 * https://github.com/elastic/kibana/blob/ec30f2aeeb10fb64b507935e558832d3ef5abfaa/x-pack/plugins/spaces/server/usage_stats/usage_stats_client.ts#L113-L118
```